builtin/logical/database/path_roles.go

```diff
@@ -181,10 +181,10 @@ func (b *databaseBackend) pathRoleCreate() framework.OperationFunc {
 }
 
 type roleEntry struct {
-	DBName     string              `json:"db_name" mapstructure:"db_name" structs:"db_name"`
-	Statements dbplugin.Statements `json:"statements" mapstructure:"statements" structs:"statements"`
-	DefaultTTL time.Duration       `json:"default_ttl" mapstructure:"default_ttl" structs:"default_ttl"`
-	MaxTTL     time.Duration       `json:"max_ttl" mapstructure:"max_ttl" structs:"max_ttl"`
+	DBName     string              `json:"db_name"`
+	Statements dbplugin.Statements `json:"statements_list"`
```

Comment:
Why "statements_list"?

Reply:
Since we are modifying dbplugins.Statements, this value must be upgraded and it needs to be stored in a new place.